packages/client/hooks/useSlackChannels.ts

```diff
@@ -13,18 +14,20 @@ const useSlackChannels = (
       const botManager = new SlackClientManager(slackAuth.botAccessToken!)
       const userManager = new SlackClientManager(slackAuth.accessToken!)
       const [channelResponse, convoResponse] = await Promise.all([
-        userManager.getChannelList(),
+        userManager.getConversationList(),
         botManager.getConversationList(['im'])
       ])
       if (!isMounted) return
       if (!channelResponse.ok) {
         console.error(channelResponse.error)
         return
       }
-      const {channels: publicChannels} = channelResponse
+      const {channels} = channelResponse
```

Comment:
we should change the return type of `getConversationList` so we don't have to recast its response.